region/norfair/east/Purple Farming Room.json

```diff
@@ -59,6 +59,28 @@
     }
   ],
   "strats": [
+    {
+      "link": [1, 1],
+      "name": "R-Mode Spark Interrupt",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        {"heatFrames": 60},
+        {"refill": ["Energy"]},
+        "h_runOverRespawningEnemies",
+        {"canShineCharge": {"usedTiles": 13, "gentleDownTiles": 1, "gentleUpTiles": 3, "openEnd": 0}},
+        {"autoReserveTrigger": {}},
+        "canRModeSparkInterrupt"
```

Comment:
This ends at 1, so it should have some heat frames.  But it also says you don't farm after proccing reserves so that may be taken care of already by the presumed post strat farm.

Reply:
Moved the end to [2]